ReSharper.FSharp/test/data/features/daemon/identifierHighlighting/Computation expressions.fs

```diff
@@ -0,0 +1,7 @@
+open System
+open System.Threading.Tasks
+
+let asyncCex = async { let! a = Task.Run(fun x -> 10) |> Async.AwaitTask
+                       a |> ignore } |> Async.RunSynchronously
```

Comment:
Could it be something simple, like the following?
```fsharp
let asyncCex = async { return 1 }
```

Reply:
Will do